server/service/integration_mdm_setup_experience_test.go

```diff
@@ -1606,685 +1603,6 @@ func (s *integrationMDMTestSuite) TestSetupExperienceWithLotsOfVPPApps() {
 	}
 }
 
-func (s *integrationMDMTestSuite) TestSetupExperienceLinuxWithSoftware() {
```

Comment:
Moved all of these to `integration_enterprise_test.go` (because Windows and Linux setup experience do not require MDM)